bindings/go/evmc/host.c

```diff
@@ -41,7 +41,7 @@ static inline void go_exported_functions_type_checks()
 {
     struct evmc_context* context = NULL;
     evmc_address* address = NULL;
-    evmc_bytes32* bytes32 = NULL;
+    evmc_bytes32 bytes32;
```

Comment:
These names `bytes32` & `uint256be` look too much as type names, it confuses me a bit every time.

Reply:
That's the intention. I have to do hard-core function pointer casts in Go, because cgo (Go's C tool) does not support `const` types. Then whenever I change the EVMC part there is no error in Go bindings.

In this "fake" function I'm using the function pointer before case and the one after with exactly the same arguments and return types so this detect almost all mismatches (because non-const types bind to const ones).